tests/jupyterhub_api/test_check_api.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
This test is intended to get the api version back to verify the Hub sanity, I'm here making a get request to "https://localhost:443/hub/api/" which which would return 200 in response. 
reference- https://jupyterhub.readthedocs.io/en/latest/_static/rest-api/index.html#path--